### PR TITLE
Change rpmbuild job to use duffy

### PR DESCRIPTION
**Pull request by johnbieren**
Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: jobs/rpmbuild.yaml [hunk 1]
```diff
@@ -1,3 +1,44 @@
+- builder:
+    name: ci-pipeline-duffy-builder
+    builders:
+        - macro-cciskel-duffy-prepared-allocate:
+            jobclass: builder
+            duffytimeoutsecs: 3600
+            playbook: sig-atomic-buildscripts/centos-ci/setup/setup-system.yml
```
Comment (p3ck) [inline jobs/rpmbuild.yaml]: You'll want to change this

**Comment by johnbieren** [comment]
Took a bunch of commits to get this tested and running properly but seems to work:
http://artifacts.ci.centos.org/fedora-atomic/rawhide/
https://ci.centos.org/job/ci-pipeline-rpmbuild/159/artifact/

File: jobs/rpmbuild_trigger.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+- job-template:
+    name: ci-pipeline-rpmbuild-trigger
+    defaults: ci-pipeline-defaults
+# TODO: trigger
```
Comment (p3ck) [inline jobs/rpmbuild_trigger.yaml]: You need to put the jms selector in here now.

Comment (johnbieren) [inline jobs/rpmbuild_trigger.yaml]: done

**Pull request merged**